endpoints/events/event.go

```diff
@@ -0,0 +1,322 @@
+package events
+
+import (
+	"context"
+	"fmt"
+	"github.com/julienschmidt/httprouter"
+	"github.com/prebid/prebid-server/analytics"
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/errortypes"
+	"github.com/prebid/prebid-server/stored_requests"
+	"time"
+
+	accountService "github.com/prebid/prebid-server/account"
+	"net/http"
+	"net/url"
+	"strconv"
+)
+
+const (
+	DefaultTimeoutMS = 1000
```

Comment:
Thank you. We may expand this to a config option in the future. I think this is a good starting point. We generously expect cache writes to take 50ms and account lookups should very likely be quick. I think this also matches well with our default auction timeout.

Reply:
I agree, I opted for expanding it now rather than later.